tests/python/test_print.py

```diff
@@ -95,6 +95,27 @@ def func(k: ti.f32):
     ti.sync()
 
 
+@ti.test()
+def test_print_list():
+    x = ti.Matrix.field(2, 3, dtype=ti.f32, shape=(2, 3))
+    y = ti.Vector.field(3, dtype=ti.f32, shape=())
+
+    @ti.kernel
+    def func(k: ti.f32):
+        w = [k, x.shape]
+        print(w + [y.n])  # [233.3, 3]
```

Comment:
nit: should this be `[233.3, [2, 3], 3]`?

Reply:
```suggestion
        print(w + [y.n])  # [233.3, [2, 3], 3]
```